src/components/footer.tsx

```
@@ -0,0 +1,214 @@
+                <li className='text-xl font-medium text-white'>
+                  <Link href='/contact-us' className='text-white'>Contact</Link>
+                </li>
+              </ul>
+              <ul>
+                <li className='text-2xl pb-2 text-white font-semibold'>
+                  SOCIAL
+                </li>
+                <li className='text-xl font-medium text-white'>
+                  <a
+                    href='https://www.linkedin.com/company/next-codez/'
+                    target='_blank'
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)